app/controllers/my-projects/assignment/recommendations/add.js

```diff
@@ -270,8 +270,17 @@ export default class MyProjectsProjectRecommendationsAddController extends Contr
       dispositionChangeset.set(targetField, recommendation);
     }
 
-    // IDs for Waiver of Recommendation
-    if (([717170002, 717170006, 717170008].includes(recommendation)) && (participantType !== 'BP')) {
+    // A selection of 'Waiver of Recommendation' makes these four vote & member fields irrelevant
+    // Borough Presidents (BP) do not have these vote & member fields at all
+    // For Borough Board, 71717002 = 'Waiver of Recommendation'
```

Comment:
Since 277-280  seems to be a duplicate of 284-287, how about combining the conditionals? Something like:
```
if (
   ([717170008].includes(recommendation) && (participantType === 'CB')) ||
   ([717170002].includes(recommendation) && (participantType === 'BB'))
) {
   ...
}
```

Reply:
Cool, I'll make the changes